aptos-move/framework/aptos-token/sources/token_event_store.move

```diff
@@ -0,0 +1,316 @@
+/// This module provides utils to add and emit new token events that are not in token.move
+module aptos_token::token_event_store {
+    use std::string::String;
+    use std::signer;
+    use aptos_framework::event::{Self, EventHandle};
+    use aptos_framework::account;
+    use std::option::Option;
+    use aptos_std::any::Any;
+    use std::option;
+
+    friend aptos_token::token;
+
+    //
+    // Collection mutation events
```

Comment:
Should we include previous description/uri/maximum, etc.

Reply:
Discussed here also https://github.com/aptos-labs/aptos-core/pull/5715#discussion_r1035192468

Reply:
seems quorum reached. I added old values.